src/scss/components/_initiatives.scss

```diff
@@ -1,16 +1,218 @@
+.workshop-image {
+	max-width: 100%;
+}
+
+.comments {
+	margin-top: rem(50);
+
+	p:first-child {
+		color: $grey-light;
+	}
+
+	.comment-name {
+		color: $black;
+		font-weight: $font-weight-medium;
+	}
+
+	.submitted-comment {
+		color: $grey-light;
+
+		.comment-name {
+			color: $grey-light;
+		}
+	}
+
+	.comment {
+		margin-bottom: rem(60);
+		padding-bottom: rem(1);
+		padding-left: 0;
+	}
+}
+
+.workshop-page .description {
+	margin-bottom: rem(100);
+
+	h3 {
+		border-bottom: rem(5) solid $gold;
+		line-height: rem(60);
+	}
+
+	p:first-child {
+		border-bottom: rem(5) solid $gold;
+		font-size: rem(24);
+		line-height: rem(60);
+	}
+}
+
+.submitted-comment-message {
+	display: none;
+	margin: auto auto;
+	text-align: center;
+
+	span {
+		background: $gold-light;
+		border-radius: rem(3);
+		display: block;
+		padding: rem(12);
+	}
+
+	.fl-syllabification-separator {
+		display: inline;
+		padding: 0;
+	}
+}
+
+#comment-form {
+	margin-bottom: rem(150);
+
+	.asterisk-def {
+		color: $grey-light;
+		display: inline;
+		font-size: rem(20);
+	}
+
+	.form-control {
```

Comment:
This selector seems attached to form fields not form controls. Please rename. Thanks.